test_pull_requests

```diff
@@ -649,15 +668,12 @@ popd
       return comment
     end
 
-    #
-    # Gets a comment with given prefix
-    #
+    # get_comment_with_prefix returns a bot comment with the given prefix, if one exists
```

Comment:
> Bot comment means a very specific comment.

unclear what you mean -- in this context this is not a special comment, just one with the prefix